src/components/content/rich-text/rich-text-query.ts

```diff
@@ -0,0 +1,96 @@
+import { graphql } from 'gatsby';
+
+export const richTextFragment = graphql`
+  fragment richText on ContentfulBlogPostContent {
```

Comment:
As you already proposed in your description, please give it a more meaningful name.

Reply:
The problem with with renaming it is, that the type is autogenerated by the plugin using the published entities of that type in contentful. I do not know how to change that behavior. I have only read that there is a possibility to hardcode to types somehow. At ADAC we also had a `schema.gql` where all types have been hardcoded.

(This would have also prevented the build fail we had with the fcb case)

Reply:
Sorry, that's not what I meant. I just meant to rename the variable and fragment name. Like this:

```
export const ContentfulRichText = graphql`
  fragment ContentfulRichText on ContentfulBlogPostContent {
    ...
```

This works without changing anything to GraphQl.

Reply:
Oh, well, now I got your point, I changed it 👍